Makefile

```diff
@@ -62,18 +62,13 @@ test-all-components: ## Run unit tests for all pipeline components
 		$(MAKE) test-components GROUP=$$(basename $$component_group) ; \
 	done
 
-sync-assets: ## Sync assets folder to GCS. Must specify pipeline=<training|prediction>
```

Comment:
We might want to leave the `assets` mechanism in place for other reasons - e.g. a custom dataflow script. WDYT?